pkgs/development/ocaml-modules/pa_ounit/default.nix

```diff
@@ -1,18 +1,18 @@
-{ lib, buildOcaml, ocaml, fetchurl, ounit }:
-
-if lib.versionAtLeast ocaml.version "4.06"
-then throw "pa_ounit is not available for OCaml ${ocaml.version}"
-else
```

Comment:
i do not know for sure if added line 14 provides the same

Reply:
It does :smile:

Reply:
or should it be `minimumSupportedOcamlVersion`? i am confused with the different namings that exist here.

Reply:
oh woops you're right. in `buildOcaml` it's `minimumSupportedOcamlVersion`
In `buildDunePackage` and `buildOasisPackage` it's `minimumOCamlVersion`

Reply:
is it desired to change this to a single variable? because all do the exact same thing.

Reply:
I wouldnt be opposed to it

Reply:
I just looked over it again. I think the old code does the exact opposite like a `maximumOCamlVersion`. so i will leave the old code and address this issue in a new PR.

Reply:
:facepalm: you're quite right

Reply:
anything to add for review?